### PR TITLE
add vagrant vm

### PR DESCRIPTION
**Pull request by chaserx**
sorta.

### CODE REVIEW

**Comment by chaserx** [comment]
currently fails. says it can't find the cookbook 'streakers'. Maybe there's a typo or I've forgotten a require somewhere

**Comment by chaserx** [comment]
maybe @mikedillion can see what I've done wrong?

File: cookbook/Berksfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+source 'https://supermarket.getchef.com'
+
+metadata
+cookbook 'golang', '~> 1.5.0'
```
Comment (mikedillion) [inline cookbook/Berksfile]: This should go in `cookbook/metadata.rb`

**Comment by mikedillion** [comment]
I think you're just missing a Berksfile in the root directory that looks like:

```
source 'https://supermarket.getchef.com'

cookbook 'streakers', path: './cookbook'
```

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    app.vm.synced_folder './', '/home/vagrant/streakers'
+    app.ssh.forward_agent = true
+
+    cache app, :apt, '/var/cache/apt/archives'
+    cache app, :chef, '/var/chef/cache'
+
+    app.vm.provision :chef_solo do |chef|
+      chef.version = :latest
+      chef.log_level = :debug
+      chef.run_list = ['recipe[streakers::development]']
```
Comment (mikedillion) [inline Vagrantfile:37]: t'ain't one of these thur yet

**Comment by mikedillion** [comment]
+1!

**Comment by chaserx** [comment]
There's something wacky about the Go installation, can't install packages.

**Comment by chaserx** [comment]
fixed

**Pull request merged**